app/controllers/investigations/ts_investigations_controller.rb

```diff
@@ -163,6 +168,11 @@ def set_repeat_step(model = :investigation)
                    end
   end
 
+  def set_risk_assessment_form
+    @risk_assessment_form = TradingStandardRiskAssessmentForm.new(current_user: current_user, investigation: @investigation, businesses: businesses_from_session, product: @product)
+    set_repeat_step(:trading_standard_risk_assessment_form)
+  end
```

Comment:
Trading Standards is always plural:
```suggestion
  def set_risk_assessment_form
    @risk_assessment_form = TradingStandardsRiskAssessmentForm.new(current_user: current_user, investigation: @investigation, businesses: businesses_from_session, product: @product)
    set_repeat_step(:trading_standards_risk_assessment_form)
  end
```